zerver/tests/test_subs.py

```diff
@@ -185,6 +184,37 @@ def test_creating_streams(self) -> None:
         for stream in existing_streams:
             self.assertTrue(stream.invite_only)
 
+    def test_invalid_stream_name(self) -> None:
+        """
+        Creating a stream with invalid 'Cc', 'Cs' and 'Cn' category of unicode characters in stream name
```

Comment:
The test only tests `Cc` and `Cn`.  Which is fine, just we shouldn't claim more than we're actually doing. :)

Reply:
Sorry for that, I removed it thank you !!